manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -235,6 +235,9 @@ data:
         metadata:
           labels:
             security.istio.io/tlsMode: {{ index .ObjectMeta.Labels `security.istio.io/tlsMode` | default "istio"  | quote }}
+            {{- if eq (index .ProxyConfig.ProxyMetadata "ISTIO_META_ENABLE_HBONE") "true" }}
+            networking.istio.io/tunnel: {{ index .ObjectMeta.Labels `networking.istio.io/tunnel` | default "h2"  | quote }}
```

Comment:
I think we mentioned this earlier, but this should not have "istio" in the name. The protocol is not istio-specific, and we should try to communicate that.

Reply:
I agree but don't have any better ideas. It needs to be a domain we own per-k8s guidelines I think.